ui/admin/tests/helpers/selectors.js

```diff
@@ -43,6 +43,8 @@ export const MODAL_WARNING_CANCEL_BTN =
 export const MODAL_WARNING_TITLE = '.hds-modal__header';
 export const MODAL_WARNING_MESSAGE = '.hds-modal__body';
 
+// Search filtering selectors
```

Comment:
None blocking comment: within the auth-method refactor, I add a [similar selector](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/tests/acceptance/auth-methods/selectors.js#L108) but at the auth-method domain level.
I knew sooner or alter we will have to make it as common, so I will refactor that to use this one! Thanks!